### DISCUSSION TITLE
Organization のランキングでContribution数の順位も欲しい

### DISCUSSION THREAD

**Discussion by yamadashy** [increments/qiita-discussions#187] [要望・アイデア・不具合]
# 概要
現在各ランキングは以下のようになっていると思います。
- タグのランキング ... 記事投稿数
- ユーザーのランキング ... Contribution数
- Organizationのランキング ... 記事投稿数

「記事の投稿数」は活発度、「Contribution数」はユーザー人気度と捉えると、
Organizationのランキングでは、「Contribution数」の順位もあるほうが良いのかなあと思いました。

# 詳細
現状は「アウトプット数の多い組織のランキング」として機能しているので、別ページとして「Contribution数の多い組織ランキング」もあると、また別の価値をユーザーに提供できると考えています。
ユーザーが求めているものはわかりませんが、「Contribution数の多い記事を投稿している企業」に関心のあるユーザーも一定数いらっしゃるのではないかと。

**Comment by kskwtnk** [comment]
@yamadashy 
ご意見ありがとうございます！

現在我々としてもOrganizationを盛り上げたいと思っており、色々画策中です
今回のランキングの意見も、議論の中に取り入れさせていただきます！

**Comment by yamadashy** [comment]
ご回答ありがとうございます。

> 現在我々としてもOrganizationを盛り上げたいと思っており、色々画策中です

そうなんですね！楽しみにしてます。
